src/test/java/stringcalculator/DelimiterParser.java

```diff
@@ -5,7 +5,10 @@
 
 public class DelimiterParser {
 
-    private final Pattern regexPattern = Pattern.compile("//(.)\n(.*)");
+    private final Pattern regexPattern = Pattern.compile(ValidationRegex.CUSTOM_DELIMITER_REGEX.getRegex());
+
+    public DelimiterParser() {
+    }
 
     public String[] parseDelimiter(String text) {
```

Comment:
```suggestion
    private String[] parsedNumbers;
    public DelimiterParser(String text) {
        this.parsedNumbers = parseDelimiter(text);
    }
   
    private String[] parseDelimiter(String text) {
```

와 같이 상태값과 초기화 할 수 있는 것을 활용해보면 어떨까요?